src/styles/blocks/card.scss

```
@@ -0,0 +1,82 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  display: flex;
+  flex-direction: column;
+  flex-shrink: 0;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: $white;
+
+  &__product-img {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    margin: 40px 16px 4px;
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
```

can be less, check other places 
```suggestion
  &__product {
   &-img {
     margin: 0 auto;
     width: 160px;
     height: 134px;
    }
    &-name {
     margin: 40px 16px 4px;
     color: $main-accent;
     font-size: 12px;
     font-weight: 500;
     line-height: 18px;
    }
  }
```